src/main/java/cs2103/v15_1j/jimjim/controller/JJController.java

```diff
@@ -37,6 +37,17 @@ public DataLists getMasterList() {
 	public DataLists getDisplayList() {
 		return states.displayList;
 	}
+	
+	@Override
+	public String getFilePath() {
+		return this.states.config.savePath;
+	}
+	
+	@Override
+	public void setFilePath(String filePath) {
```

Comment:
should create a new SaveLocationCommand and return the feedback to UI to display if there are any errors.